presto-docs/src/main/sphinx/sql/create-view.rst

```diff
@@ -38,6 +38,9 @@ In the ``INVOKER`` security mode, tables referenced in the view are
 accessed using the permissions of the query user (the *invoker* of the
 view). A view created in this mode is simply a stored query.
 
+The ``default-view-security-mode`` can be used to configure the default
+security mode for view creation.
+
```

Comment:
Consider adding an example that shows how to use ``default-view-security-mode``.

Reply:
@steveburnett It can be used as any other session property. Do you have anything specific in mind?

Reply:
The example you added works, thanks!